src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -22,7 +22,20 @@ import com.google.cloud.pubsub.v1.Publisher
 import com.google.protobuf.ByteString
 import com.google.pubsub.v1.PubsubMessage
 import com.google.pubsub.v1.TopicName
+import com.xenomachina.argparser.ArgParser
 import org.slf4j.LoggerFactory
+import java.io.File
+
+class PublisherArgs(parser: ArgParser) {
```

Comment:
CmdLineClient is more appropriate place for these classes, no?

Reply:
Done